tests/bud.bats

```diff
@@ -3981,6 +3981,36 @@ _EOF
   run_buildah rmi myalpine
 }
 
+@test "build verify cache behaviour with --cache-ttl" {
+  mkdir -p ${TEST_SCRATCH_DIR}/bud/platform
+
+  cat > ${TEST_SCRATCH_DIR}/bud/platform/Dockerfile1 << _EOF
+FROM alpine
+RUN touch hello
+RUN echo world
+_EOF
+
+  run_buildah build $WITH_POLICY_JSON --layers -t source -f ${TEST_SCRATCH_DIR}/bud/platform/Dockerfile1
+  # Sleep for 1s to verify that we are not
+  # going to use cached layers.
+  sleep 1s
+  # Specify --cache-ttl 0.5s and cache should
+  # not be used since cached image is 1s old.
+  run_buildah build $WITH_POLICY_JSON --cache-ttl=0.5s --layers -t source -f ${TEST_SCRATCH_DIR}/bud/platform/Dockerfile1
+  # Should not contain `Using cache` since all
+  # cached layers are 1s old.
+  assert "$output" !~ "Using cache"
+  # clean all images and cache
+  run_buildah rmi --all -f
```

Comment:
This deletes the base image as well.  Call `_prefetch alpine` to restore a copy from a cache and make this test faster.